.github/workflows/macos.yml

```diff
@@ -123,9 +123,10 @@ jobs:
         fetch-tags: true
         persist-credentials: false
 
-    - uses: quansight-labs/setup-python@b9ab292c751a42bcd2bb465b7fa202ea2c3f5796 # v5.3.1
+    - uses: astral-sh/setup-uv@b5f58b2abc5763ade55e4e9d0fe52cd1ff7979ca
```

Comment:
It looks like this is picking up the wrong Python as well, it shows `python-3.13` and not `python-3.13t`, and the file names for extensions contain `cpython-313-darwin`.

Reply:
I thought I had double-checked this stuff but you are much better at catching this sort of thing in CI logs. Thanks so much for giving this a close look.